browser/sources.gni

```diff
@@ -119,79 +99,107 @@ brave_chrome_browser_deps = [
   "//brave/browser/themes",
   "//brave/browser/translate/buildflags",
   "//brave/browser/ui",
-  "//brave/common",
+  "//brave/browser/ui/bookmark",
   "//brave/common:pref_names",
+  "//brave/common:switches",
+  "//brave/common",
   "//brave/components/binance/browser/buildflags",
   "//brave/components/brave_ads/browser/buildflags",
   "//brave/components/brave_component_updater/browser",
-  "//brave/components/brave_drm",
   "//brave/components/brave_referrals/buildflags",
+  "//brave/components/brave_rewards/browser",
+  "//brave/components/brave_rewards/common",
   "//brave/components/brave_search/browser",
   "//brave/components/brave_search/common:mojom",
   "//brave/components/brave_shields/browser",
   "//brave/components/brave_shields/common",
-  "//brave/components/brave_sync",
-  "//brave/components/brave_together/browser",
   "//brave/components/brave_wallet/common/buildflags",
   "//brave/components/brave_wayback_machine:buildflags",
   "//brave/components/brave_webtorrent/browser/buildflags",
-  "//brave/components/content_settings/core/browser",
   "//brave/components/cosmetic_filters/browser",
   "//brave/components/cosmetic_filters/common:mojom",
   "//brave/components/crypto_dot_com/browser/buildflags",
+  "//brave/components/crypto_dot_com/browser",
```

Comment:
shouldn't we add this only if the buildflag is enabled? also, this and similar entries are injected below with `brave_chrome_browser_deps += brave_browser_crypto_dot_com_deps`
`

Reply:
Hmm... I think you're right, looks like I need to make a few other adjustments here, will adapt + fix other platforms (ios / Mac) and will ping you back. Thanks for the review!